packages/v4/src/content/design-guidelines/components/dropdown/dropdown.md

```diff
@@ -2,4 +2,39 @@
 id: Dropdown
 section: components
 ---
-Use a **dropdown** when you want to present a list of actions in a limited space.
+
+A **dropdown** component is used to present a menu of actions or links in a constrained space that will trigger a process or navigate to a new location. See Select component [link] if you want to make one or more selections from a list of items in a value list.
```

Comment:
Need to edit the "select component [link]" to:
"See [Select component](/components/select/) if you want..."

Reply:
Also change beginning to "A **Dropdown** is used to..." - so capitalize dropdown, and remove the word "component". This is just so it's more in line with the start of other guidelines!